project/Dependencies.scala

```diff
@@ -19,14 +19,15 @@ object Dependencies {
     crossScalaVersions := Seq("2.11.8"), // "2.12.0-M4"
     scalaVersion := crossScalaVersions.value.head,
     scalaStmVersion := sys.props.get("akka.build.scalaStmVersion").getOrElse("0.7"),
-    scalaCheckVersion := sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.11.6"),
+    scalaCheckVersion := sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.13.2"),
     scalaTestVersion := {
       scalaVersion.value match {
         case "2.12.0-M1" => "2.2.5-M1"
         case "2.12.0-M2" => "2.2.5-M2"
         case "2.12.0-M3" => "2.2.5-M3"
         case "2.12.0-M4" => "2.2.6"
-        case "2.12.0-M5" => "3.0.0-RC4"
+        case "2.12.0-M5" => "3.0.0"
+        case "2.11.8"  => "3.0.0"
```

Comment:
right now that is what we want, but it is likely that this will continue to diverge for different major scala versions so I would suggest 

```
scalaVersion.value match {
  case "2.12.0-M5" => "3.0.0"
  case _ => 3.0.0"
}
```